<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScpToolGui</class>
 <widget class="QWidget" name="ScpToolGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScpGuiTool </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>SCP GUI Tool</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="text">
        <string>远程信息：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="userLineEdit">
       <property name="toolTip">
        <string>远程主机用户名</string>
       </property>
       <property name="placeholderText">
        <string>用户名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>@</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ipLineEdit">
       <property name="toolTip">
        <string>远程主机IP地址</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="passwdLabel">
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="toolTip">
        <string>远程主机密码</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="sendTab">
      <attribute name="title">
       <string>发送文件</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="localFileLabel">
           <property name="text">
            <string>待发送文件：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="localFileLineEdit">
           <property name="toolTip">
            <string>本地文件的路径</string>
           </property>
           <property name="placeholderText">
            <string>本地路径</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="localFileButton">
           <property name="toolTip">
            <string>本地文件的路径</string>
           </property>
           <property name="text">
            <string>选择</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="remoteSaveLabel">
           <property name="text">
            <string>远程保存路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="remoteSaveLineEdit">
           <property name="toolTip">
            <string>远程主机的保存路径</string>
           </property>
           <property name="placeholderText">
            <string>远程保存路径</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="receiveTab">
      <attribute name="title">
       <string>接收文件</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="remoteReceiveFileLabel">
           <property name="text">
            <string>待接收文件：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="remoteFileLineEdit">
           <property name="toolTip">
            <string>远程主机的文件路径</string>
           </property>
           <property name="placeholderText">
            <string>远程文件路径</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="localSaveLabel">
           <property name="text">
            <string>保存路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="localSaveLineEdit">
           <property name="toolTip">
            <string>本地的保存路径</string>
           </property>
           <property name="placeholderText">
            <string>本地保存路径</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="localSaveButton">
           <property name="toolTip">
            <string>本地的保存路径</string>
           </property>
           <property name="text">
            <string>选择</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
